code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -0,0 +1,212 @@
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "wolf's bane"
+	overdose_threshold = 35
+	reagent_state = SOLID
+
+/datum/reagent/medicine/C2/arnica/on_mob_life(mob/living/carbon/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.bleed_rate)
+			H.adjustBruteLoss(-0.5*H.bleed_rate*REM)
+			H.bleed_rate += 2
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/arnica/overdose_process(mob/living/carbon/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		H.bleed_rate += 2
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/acetaminophen
+	name = "Acetaminophen"
+	description = "A common pain reliever. Does minor liver damage."
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "a common spacehold drug"
+	reagent_state = SOLID
+
+/datum/reagent/medicine/C2/acetaminophen/on_mob_life(mob/living/carbon/M)
+	M.adjustLiverLoss(1*REM)
+	M.adjustBruteLoss(-1*REM)
+	..()
+	return TRUE
+
+/******BURN******/
+
+/datum/reagent/medicine/C2/silver_sulfadiazine
+	name = "Silver Sulfadiazine"
+	description = "Used to treat serious burns. Scales with prolonged exposure, but causes burns to itch as well."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	var/resetting_probability = 0
+	var/spammer = 0
+
+/datum/reagent/medicine/C2/silver_sulfadiazine/on_mob_life(mob/living/carbon/M)
+	M.adjustFireLoss(-0.25*current_cycle*REM)
+	if(prob(resetting_probability))
+		if(spammer < world.time)
+			to_chat(M,"<span class='warning'>You can't help but to itch the burn.</span>")
+			spammer = world.time + (10 SECONDS)
+		var/scab = rand(1,7)
+		M.adjustBruteLoss(scab*REM)
+		M.bleed(scab)
+		resetting_probability = 0
+	resetting_probability += (5*(current_cycle/10)) // 10 iterations = >51% to itch
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/neomycin_sulfate
+	name = "Neomycin Sulfate"
+	description = "Used to treat burns. Does minor eye damage."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	var/resetting_probability = 0
+	var/message_cd = 0
+
+/datum/reagent/medicine/C2/neomycin_sulfate/on_mob_life(mob/living/carbon/M)
+	var/obj/item/organ/eyes/the_hills_have = M.getorganslot(ORGAN_SLOT_EYES)
+	M.adjustFireLoss(-1*REM)
+	the_hills_have?.applyOrganDamage(1)
+	..()
+	return TRUE
+
+/******OXY******/
+#define PERF_BASE_DAMAGE		0.5
+
+/datum/reagent/medicine/C2/perfluorodecalin
+	name = "Perfluorodecalin"
+	description = "Restores oxygen deprivation while producing a lesser amount of toxic byproducts. Both scale with exposure to the drug and current amount of oxygen deprivation. Overdose causes toxic byproducts regardless of oxygen deprivation."
+	reagent_state = LIQUID
+	color = "#FF6464"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	overdose_threshold = 35 // at least 2 full syringes +some, this stuff is nasty if left in for long
+
+/datum/reagent/medicine/C2/perfluorodecalin/on_mob_life(mob/living/carbon/human/M)
+	var/oxycalc = 2.5*REM*current_cycle
```

Comment:
With inoculation beforehand and regular doses to keep the reagent amount up but less than overdose, this will result in players becoming literally immune to any form of oxyloss, immediately trading 100% of it for 25% toxin damage. This is, of course, not _necessarily_ a problem, but I do think it should be brought up.

Reply:
~~Perf's~~ Convermol base damage now scales with current_cycle so eventually you're going to get to a point where you're taking too much consistent tox damage.

I think this interaction on top of multiver's nerf will fix the issue.